src/components/home/reducers.js

```diff
@@ -0,0 +1,28 @@
+import { REQUEST_SEARCH, UPDATE_SEARCH } from './actionTypes';
+const initialState = {
+    isloading: false,
+    results: [],
+};
+
+export default (state = { ...initialState }, action) => {
+    switch (action.type) {
+        case REQUEST_SEARCH: {
+            const { payload } = action;
+            return {
+                ...state,
+                isloading: true,
+                payload,
```

Comment:
no this is wrong, there should not be a `payload` key on the state. What is payload?

Reply:
i don't even need this piece of the state :)